### PR TITLE
New branch

### PR DESCRIPTION
**Pull request by Dave-2121**
A simple pull request!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,40 +1 @@
...
-- [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
-
-You can check out [the Next.js GitHub repository](https://github.com/vercel/next.js/) - your feedback and contributions are welcome!
-
-## Deploy on Vercel
-
-The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
-
-Check out our [Next.js deployment documentation](https://nextjs.org/docs/deployment) for more details.
+hi this a readme!! blah blahh blah
```
Comment (Bhaikko-01) [inline README.md:1]:
```suggestion
Hi this is a README!!
```

File: pages/index.tsx [hunk 2]
```diff
@@ -1,118 +1,12 @@
...
-            </span>
-          </h2>
-          <p className={`m-0 max-w-[30ch] text-sm opacity-50 text-balance`}>
-            Instantly deploy your Next.js site to a shareable URL with Vercel.
-          </p>
-        </a>
-      </div>
-    </main>
+    <>
+      <div>hello</div>
```
Comment (Bhaikko-01) [inline pages/index.tsx:8]: Capitalize this

File: styles/globals.css [hunk 3]
```diff
@@ -1,33 +1,3 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
```
Comment (Bhaikko-01) [inline styles/globals.css:3]: Sample comment

**Review by Bhaikko-01 — CHANGES_REQUESTED** [review]

**Review by Dave-2121 — COMMENTED** [review]
!!!!

**Pull request merged**